src/schema/metadata/PowerLineFrequency.yaml

```diff
@@ -0,0 +1,11 @@
+---
+key_name: PowerLineFrequency
+description: |
+  Frequency (in Hz) of the power grid at the geographical location of the MEG
+  instrument (for example, 50 or 60).
+anyOf:
+  - type: number
+    exclusiveMinimum: 0
+    unit: Hz
+  - type: string
+    pattern: ^n/a$
```

Comment:
Could we replace the `^n/a$` pattern with a one-item `enum`?

Reply:
yes, technically that would be equivalent except for the message that a user gets upon validating that schema. (I.e., "foo must be pattern X" versus "foo must be one of X" or the like). But that small difference would be no problem IMHO.

Reply:
The current issue is that the tables render these kinds as `object or string` instead of `object or "n/a"`. I thought maybe checking against an enum list and rendering those values directly if the list is short would be easier than figuring something out for pattern strings, but maybe it'll be easier if I figure out what the characters in the pattern represent, and what patterns are possible?

Reply:
> maybe it'll be easier if I figure out what the characters in the pattern represent, and what patterns are possible?

There are some libraries that allow you to generate strings from a given regular expression, see: https://github.com/asciimoo/exrex

So perhaps it wouldn't even be too difficult. And coincidentally, it may provide us with a nice double check of our regexps :-)